src/riakc_ts_query_operator.erl

```diff
@@ -48,7 +47,12 @@ serialize_interpolations([{Key, Value} | RemainingInterps],
     UpdatedInterps = [#rpbpair{key=Key, value=Value} | SerializedInterps],
     serialize_interpolations(RemainingInterps, UpdatedInterps).
 
-deserialize({error, Message}) -> {error, Message};
+deserialize({error, {Code, Message}}) when is_integer(Code), is_list(Message) ->
```

Comment:
This is not necessary, for the reason that incoming messages don't ever contain strings (if the server sends any strings, the encoder will blow up somewhere and the message will not be delivered).